fe/fe-core/src/main/java/com/starrocks/transaction/TransactionStateListener.java

```diff
@@ -20,6 +20,8 @@
 // TransactionStateListener will be created by the FE master process before it commit a transaction.
 // Used to check if a transaction can be committed and save some information in the TransactionState.
 public interface TransactionStateListener {
+    String getTableName();
```

Comment:
If we support multi-tables transactions, what will this interface be. Or when users use broker load to ingest multi tables, what will this function return?

Reply:
Multiple `TransactionStateListener` will be created in this case.